pkg/daemon/daemon.go

```diff
@@ -526,7 +526,7 @@ func (dn *Daemon) isDesiredMachineState() (bool, string, error) {
 	// config, the machine is definitely not in its desired state. this function
 	// will return true (meaning they are reconcilable) if there aren't actually
 	// changes.
-	reconcilableError, err := dn.reconcilable(currentConfig, desiredConfig)
+	reconcilableError := dn.reconcilable(currentConfig, desiredConfig)
 	if err != nil {
```

Comment:
Probably remove this error check as well

Reply:
thanks! will do!